console/console-init/ui/src/modules/iot-device-detail/components/ConfigurationInfo/ConfigurationInfo.tsx

```diff
@@ -149,8 +150,9 @@ export const ConfigurationInfo: React.FC<IConfigurationInfoProps> = ({
                   <Card>
                     <CardBody>
                       <DropdownWithToggle
-                        id="ci-credential-type-dropdown"
-                        toggleId={"ci-credential-type-dropdown"}
+                        id="config-info-credential-type-dropdowntoggle"
```

Comment:
The id and toggleIDs for `DropdownWithToggle` component are vaguely named, any reason for that?

Reply:
@ajaypratap003 Could you please suggest a good name for id and toggleId in this case?

Reply:
I would add some information about where are we currently so probably 'device-config-info-credential-type-dropdowntoggle'. Some better idea @ajaypratap003 ?

Reply:
@obabec This page belongs to device details page in which we have a  configuration info tab and DropdownToggle is used for credentials filter. First dopdown have some options like all credentials, enabled, password etc. and scond one will have the options based on first dropdown selection.
I would suggest id here
` id='device-detail-config-info-credential-type-dropdown'` 
` toggleId='device-detail-config-info-credential-type-dropdowntoggle'` 
@obabec wdyt?

![image](https://user-images.githubusercontent.com/11775081/88267232-2c8de180-ccee-11ea-83d0-fb20528ce94a.png)